extension-androidauto/build.gradle.kts

```diff
@@ -22,11 +22,12 @@ android {
 }
 
 dependencies {
-  compileOnly(project(":sdk"))
+  implementation(project(":sdk"))
```

Comment:
It was made compileOnly to be used as a true "extension" that user have to specify the maps sdk themselves in the build.gradle.

Reply:
OK updating after running `r21e` `r22` ci configurations

Reply:
unit tests fail if this is added by itself. noticed the other extensions use `implementation`

also, was running into failures with `testCompile`. So i've added `testImplementation`

Reply:
sounds good!